lib/Runtime/Provisioner/Provisioner.cpp

```diff
@@ -101,11 +101,14 @@ llvm::Error Provisioner::provision(DAGListTy &networks, Module &module,
       // compiled the function reuse it.
       auto it = functions_.find(node->name);
       if (it == functions_.end()) {
+        // Copy BackendOptions and add the compiler hints for this function.
+        auto options = cctx.backendOpts;
+        options.backendHints = node->backendHints;
```

Comment:
Putting this into the BackendOptions really makes me think I can configure it in the CompilationContext, but it is always overridden here. Can we be clearer about this? Maybe log an error if this is set already?

Reply:
Error log added.